docs/upgrade/upgrade-security.asciidoc

```diff
@@ -1,44 +1,75 @@
 [chapter]
 [[upgrade-intro]]
 
-
 = Upgrade {elastic-sec}
 
 We highly recommend that all {elastic-sec} users keep their deployments up to date with the latest available {stack} version to access new features, security updates, and performance enhancements.
 
 For upgrading to the latest {stack} version, we recommend using the {kibana-ref}/upgrade-assistant.html[Kibana Upgrade Assistant].
 
+NOTE: For detailed information about the 8.0.0 release, check out the <<release-notes-8.0.0, Release notes>>.
+
 
 [discrete]
 [[pre-upgrade-security]]
-== Pre-upgrade requirements for 8.0
+== Pre-upgrade requirements
+
+*Track rules that are automatically disabled when upgrading*
```

Comment:
This section should match the section in this [other page](https://security-docs_1558.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.0.0.html) (@nastasha-solomon's PR is [here](https://github.com/elastic/security-docs/pull/1558) for reference):

<img width="800" alt="image" src="https://user-images.githubusercontent.com/616158/154387317-1a4cc00e-fd47-43a4-8b50-058d94710e3c.png">